tools/tune/multiband_drc/crossover_gen_quant_coefs.m

```diff
@@ -0,0 +1,62 @@
+function crossover_coefs = crossover_gen_quant_coefs(num_bands, sample_rate,
+						     fc_low, fc_med, fc_high);
+
+% De-normalize cutoff frequencies in respect to nyquist (half of sample rate)
+fc_low = fc_low * sample_rate / 2;
+fc_med = fc_med * sample_rate / 2;
+fc_high = fc_high * sample_rate / 2;
+
+addpath ./../crossover
+
+filter_len = 3;
+crossover.lp = cell(1:filter_len);
+crossover.hp = cell(1:filter_len);
+% Generate zeros, poles and gain for crossover with the given frequencies
+% Extend the length of lp and hp to 3 (filter_len) by flat_filter if necessary
+if (num_bands == 1)
+	% Pass-through
+	crossover.lp = [flat_filter() flat_filter() flat_filter()];
```

Comment:
isn't there a special config to put the crossover into passthrough mode?

Reply:
The passthrough mode of a component is enabled when there is no config data on *_prepare. And for multiband_drc, the passthrough mode passes through the whole processing of DRC.
The crossover pass-through here is to set num_bands to 1, then there should be no crossover on the path.